lisa/features/infiniband.py

```diff
@@ -224,11 +223,24 @@ def setup_rdma(self) -> None:
             else:
                 redhat_required_packages.append("python3-devel")
                 redhat_required_packages.append("python2-devel")
+
             node.os.install_packages(list(redhat_required_packages))
-        elif isinstance(node.os, Ubuntu):
+
+            try:
+                node.os.install_packages("kernel-devel-$(uname -r)")
+            except MissingPackagesException:
+                node.log.debug(
+                    "kernel-devel-$(uname -r) not found. Trying kernel-devel"
+                )
+                node.os.install_packages("kernel-devel")
+        elif isinstance(node.os, Ubuntu) and node.os.information.version >= "18.4.0":
```

Comment:
Is Debian OS class not supported?

Reply:
At this time, I have only implemented images supported by the [HPC team](https://github.com/Azure/azhpc-images)